Gaijinizer patch/Map299.txt

```diff
@@ -27,10 +27,10 @@ strength of the combat extra people.#GT
 
 他の部隊も、似たような状況だろう。
 > CONTEXT: Map299/events/13/pages/0/17/Dialogue 
-Two of my troops, 10 of whom we were leading
-They were committed and captivated by them.
+2 of my troops and 10 evacuating civilians were
+raped and carried off by them.
 
-Other units will have similar situations.#GT
+Other units are surely facing similar problems.#Amaboye
 > END STRING
```

Comment:
Minor rewording for flow:
```
2 of my troops and 10 of the civilians we were
escorting were raped then carried off by them.

I'm sure the other units are facing the same problems.

I'm sure the other units are facing the same problem.
```

(this is a Dialogue, so we have longer line limits)